dashboard/settings.py

```diff
@@ -285,3 +291,5 @@
     AUTHENTICATION_BACKENDS += ('django.contrib.auth.backends.ModelBackend',)
     LOGIN_REDIRECT_URL = '/'
     
+
+SETTINGS_EXPORT = ['LOGIN_URL','LOGOUT_URL','DEBUG']
```

Comment:
What's the purpose of this setting?

Reply:
I see, it is used in base.html.

Reply:
Yeah I can add a comment here, it's for the settings module. Basically every setting we want to have available to in the templates needs to be in here. I added DEBUG as I figured we'd want that.